### ISSUE TITLE
Tid_lua or Tid_wikitext

### ISSUE THREAD

**Issue by desb42** [gnosygnu/xowa#336]
Inside Xow_page_tid.java the decision to return Tid_lua or Tid_wikitext in function Identify for a Module is determined if the title ends with '/doc'

Unfortunately this needs more thought.
For dewiki the test should be for '/Doku' and for enwiktionary it should be '/documentation'
I have not looked at others (yet)

The crude fix is to test for all three (however there may be others)

Another way is by using the message string `scribunto-doc-page-name` (and any MediaWiki:scribunto-doc-page-name override)
This give a message of the form `Module:~{0}/doc` (from en.gfs) and the '/doc' can be stripped out

Yet another way is during the build phase
Each <page> has a <format> which most of the time is `text/x-wiki`
For Module pages it is mostly `text/plain` for lua code
However it is `text/x-wiki` for documentation pages
This could be used to determine the extension and add as an additional parameter to xowa_cfg

What do you think?

**Comment by gnosygnu** [comment]
Yeah, I think the best way is probably with ```scribunto-doc-page-name```. I'll handle this when I handle #327 . Let me know if you think this should be higher priority. Thanks.
